_inc/client/plans/style.scss

```diff
@@ -221,3 +221,20 @@ $plan-features-sidebar-width: 272px;
 .plans-mobile-notice.dops-card h2 {
 	margin-top: 0;
 }
+.plan-grind-skeletons {
+	display:flex;
+	margin: 0 -10px;
+
+	@include breakpoint( "<1040px" ) {
+		margin: 0 -1px;
+	}
+}
+.plan-grind-skeletons__plan {
+	flex: 1;
+	height: 400px;
```

Comment:
I wonder how we came up with this height. Can we make it the same or close to the same with the actual plan cards height?

Reply:
The height of the original is about 800px which resulted in something that didn't look quite right to me. Because you end up with these really large pulsating rectangles that take up most of the screen.

Reply:
Thanks for confirming 👍 Let's keep it as-is for now.